real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/RBSMNode.java

```diff
@@ -141,6 +143,27 @@ public int peel(byte[] buffer, int start, int bytes_available) {
 			messagePub_pot.publish(new SteeringMeasurement(-(potValue + OFFSET)/ARD_TO_DEG));
 		}
 		
+		if(message.getHeaderByte() == RBSerialMessage.RBSM_MID_ERROR){
+			switch(message.getDataWord()){
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [A switch statement does not contain a break](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/1362898413/issues/source?bid=2938665&fileBranchId=2957480#l147)